tools/ClassGen/NodeGen.cpp

```diff
@@ -415,6 +444,100 @@ int main(int argc, char **argv) {
       .dataParallel()
       .setDocstring("Performs elementwise pow(LHS, RHS).");
 
+  BB.newNode("And")
+      .addInput("LHS")
+      .addInput("RHS")
+      .addResultFromCtorArg()
+      .dataParallel()
+      .setDocstring("Performs an element-wise logical AND between the LHS and "
+                    "RHS operands.");
+
+  BB.newNode("Or")
+      .addInput("LHS")
+      .addInput("RHS")
+      .addResultFromCtorArg()
+      .dataParallel()
+      .setDocstring("Performs an element-wise logical OR between the LHS and "
+                    "RHS operands.");
+
+  BB.newNode("Xor")
+      .addInput("LHS")
+      .addInput("RHS")
+      .addResultFromCtorArg()
+      .dataParallel()
+      .setDocstring("Performs an element-wise logical XOR between the LHS and "
+                    "RHS operands.");
+
+  BB.newNode("Not")
+      .addInput("Input")
+      .addResultFromCtorArg()
+      .dataParallel()
+      .setDocstring("Performs an element-wise logical NOT of the Input "
+                    "operand.");
+
+  BB.newNode("Neg")
+      .addInput("Input")
+      .addResultFromCtorArg()
+      .dataParallel()
+      .setDocstring("Performs an element-wise negation (sign flip) of the "
+                    "Input operand.");
+
+  BB.newNode("Abs")
+      .addInput("Input")
+      .addResultFromCtorArg()
+      .dataParallel()
+      .setDocstring("Performs an element-wise ABS(x) of the Input operand.");
+
+  BB.newNode("Floor")
+      .addInput("Input")
+      .addResultFromCtorArg()
+      .dataParallel()
+      .setDocstring("Performs an element-wise FLOOR(x) of the Input operand.");
+
+  BB.newNode("Ceil")
+      .addInput("Input")
+      .addResultFromCtorArg()
+      .dataParallel()
+      .setDocstring("Performs an element-wise CEIL(x) of the Input operand.");
+
+  BB.newNode("Round")
```

Comment:
In principle no because all the frameworks (ONNX, Numpy, TensorFlow) use the Round with same rounding mode (round to even meaning that values with fractional part 0.5 are rounded to nearest even integer):
https://www.tensorflow.org/api_docs/python/tf/math/round
https://github.com/onnx/onnx/blob/master/docs/Operators.md#Round

But realizing this I actually remembered that my unit tests did not properly verified the rounding. I updated the unit tests and also replaced the **std::round** with **std::nearbyintf** which uses the correct rounding.